cg/services/orders/submitters/order_submitter.py

```diff
@@ -34,14 +33,13 @@ def store_order(self, order_in: OrderIn):
         pass
 
     @staticmethod
-    def _fill_in_sample_ids(samples: list[dict], lims_map: dict, id_key: str = "internal_id"):
+    def _fill_in_sample_ids(samples: list[Sample], lims_map: dict):
```

Comment:
The `Sample` is only used for new samples in the order, meaning that all samples in the input will miss their lims id. So we should be able to remove the check if the sample already has one set